packages/url/src/get-query-args.js

```diff
@@ -1,6 +1,7 @@
 /**
  * Internal dependencies
  */
+import { safeDecodeURIComponent } from '.';
```

Comment:
Does it make sense to import this from its actual module, rather than hopping through the package re-export? In some scenarios, such imports could lead to circular dependency issues.

```suggestion
import { safeDecodeURIComponent } from './safe-decode-uri-component';
```